MessengerOnFileForLinux/DisplayChat/src/NcursesPrintToMainWindowOperationWrapper.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <string>
+
+#include "NcursesPrintOperationWrapper.hpp"
+
+namespace
+{
+static constexpr auto hello = "Hello ";
+static constexpr auto enterPassword = "Enter the password : ";
+static constexpr auto informationToDisplayInMainWindow = "You are currently logged as ";
+static constexpr auto informationToDisplayDuringSignIn = ". If you want use messenger, you need to log in.";
+static constexpr auto informationToDisplayDuringRegistration =
+        ". You are using messenger first time. You need create account. Enter your password twice.";
+
+}//namespace
+
+
+class NcursesPrintToMainWindowOperationWrapper : public NcursesPrintOperationWrapper
+{
+public:
+    void printInformationIntoMainWindow(const std::string&) const override;
```

Comment:
przyjmujemy konwencję z nazwą argumentów w .hpp

Reply:
Niech tak będzie. Chociaż w tym przypadku jak dla mnie jeśli funkcja jest dobrze nazwana, to nazwa argumentu kompletnie nic nie wnosi. No ale mamy więcej przypadków gdzie tak nie jeste :P